emulator/periph/src/i3c.rs

```diff
@@ -179,8 +184,9 @@ impl I3c {
             }
             // we only need the first byte, which is the MDB.
             // TODO: handle more than the MDB?
+            // Drain IBI descriptor size + 4 bytes (MDB)
             self.i3c_target.send_ibi(self.tti_ibi_buffer[4]);
-            self.tti_ibi_buffer.drain(0..len + 4);
+            self.tti_ibi_buffer.drain(0..8);
```

Comment:
Theoretically this length could be more, so maybe `(len + 4).next_multiple_of(4)`?

Reply:
That sounds correct! Changed!